File: Jack.Presentation/Controllers/MVC/FamiliaCriancaController.vb
Imports System.Web.Mvc

Namespace Controllers.MVC
    Public Class FamiliaCriancaController
        Inherits Controller

        ' GET: FamiliaCrianca
        Function Index() As ActionResult
            Return View()
        End Function

        Function Representante() As ActionResult
            Return View()
        End Function

    End Class
End Namespace

File: Jack.Presentation.Web/Controllers/MVC/FamiliaCriancaController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web.Mvc;
using Application = Jack.Application;
using Model = Jack.Model;


namespace Controllers.MVC
{
	public class FamiliaCriancaController : Controller
	{

		// GET: FamiliaCrianca
		public ActionResult Index()
		{
			return View();
		}

		public ActionResult Representante()
		{
			return View();
		}

	}
}

